fdbserver/storageserver.actor.cpp

```diff
@@ -1090,100 +1144,105 @@ ACTOR Future<Void> getValueQ( StorageServer* data, GetValueRequest req ) {
 // Pessimistic estimate the number of overhead bytes used by each
 // watch. Watch key references are stored in an AsyncMap<Key,bool>, and actors
 // must be kept alive until the watch is finished.
-static constexpr size_t WATCH_OVERHEAD_BYTES = 1000;
+extern size_t WATCH_OVERHEAD_WATCHQ, WATCH_OVERHEAD_WATCHIMPL;
```

Comment:
```suggestion
size_t WATCH_OVERHEAD_WATCHQ, WATCH_OVERHEAD_WATCHIMPL;
```
This is not `extern` - it's literally in the same file ;)

Reply:
So the issue is i need to declare these variables and then initialize them towards the end of the file (since the actors aren't created yet). C++ doesn't allow me to reinitialize variables in global scope so if I were to remove the extern I would need to initialize these within a function (which I don't think makes much sense). If you have any other ides on how to do this I would be open to it

Reply:
ah right I misremembered this (I think you could do this is C but not in C++). I think the clean C++ way would be to calculate this value in a templated constexpr function (or maybe even a templated `using`-expression could work -- because when you define something as a template you get a better linker...). But I think this wouldn't really make the code more readable :)